<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_marketplacesTablePage">
       <attribute name="title">
        <string>База данных</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <widget class="QTableView" name="m_tableView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_0">
          <item row="0" column="1">
           <widget class="QLabel" name="lblChoicesTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Название таблицы:</string>
            </property>
            <property name="buddy">
             <cstring>m_cbChoicesTable</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QStackedWidget" name="m_swInputsData">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="m_pgMarketplaces">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="lblFloor">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Этаж:</string>
                </property>
                <property name="buddy">
                 <cstring>m_sbFloor</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="m_sbFloor">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblArea">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Площадь:</string>
                </property>
                <property name="buddy">
                 <cstring>m_dsbArea</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="m_dsbArea">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="suffix">
                 <string> м.</string>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000000.998999999952503</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lblAirConditioner">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Наличие &amp;кондиционера:</string>
                </property>
                <property name="buddy">
                 <cstring>m_cbAirConditioner</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="m_cbAirConditioner">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lblCostPerDay">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Стоимость аренды в день:</string>
                </property>
                <property name="buddy">
                 <cstring>m_dsbCostPerDay</cstring>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="m_dsbCostPerDay">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="suffix">
                 <string> ₽</string>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1000000.998999999952503</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="m_pgCustomers">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="lblName">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Название:</string>
                </property>
                <property name="buddy">
                 <cstring>m_leName</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="ColouredLineEdit" name="m_leName">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maxLength">
                 <number>50</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblInvoiceDetails">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Реквизиты:</string>
                </property>
                <property name="buddy">
                 <cstring>m_leInvoiceDetails</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="ColouredLineEdit" name="m_leInvoiceDetails">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lblAddress">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Адрес:</string>
                </property>
                <property name="buddy">
                 <cstring>m_leAddress</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="ColouredLineEdit" name="m_leAddress">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lblPhone">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Телефон:</string>
                </property>
                <property name="buddy">
                 <cstring>m_lePhone</cstring>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="ColouredLineEdit" name="m_lePhone">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="inputMask">
                 <string>+7(999)999-99-99;_</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="lblContactPerson">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Контактное &amp;лицо:</string>
                </property>
                <property name="buddy">
                 <cstring>m_leContactPerson</cstring>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="ColouredLineEdit" name="m_leContactPerson">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maxLength">
                 <number>255</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="m_pgRental">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="lblMarketplace_id">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Код торговой &amp;точки:</string>
                </property>
                <property name="buddy">
                 <cstring>m_cbMarketplace_id</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="m_cbMarketplace_id">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblCustomer_id">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Код &amp;клиента:</string>
                </property>
                <property name="buddy">
                 <cstring>m_cbCustomer_id</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="m_cbCustomer_id">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lblStartDate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Дата &amp;начала:</string>
                </property>
                <property name="buddy">
                 <cstring>m_leStartDate</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="ColouredLineEdit" name="m_leStartDate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="inputMask">
                 <string>00.00.0000 (день.месяц.год);_</string>
                </property>
                <property name="text">
                 <string>.. (день.месяц.год)</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lblEndDate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Дата о&amp;кончания:</string>
                </property>
                <property name="buddy">
                 <cstring>m_leEndDate</cstring>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="ColouredLineEdit" name="m_leEndDate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="inputMask">
                 <string>00.00.0000 (день.месяц.год);_</string>
                </property>
                <property name="text">
                 <string>.. (день.месяц.год)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_0">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>44</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="m_cbChoicesTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Торговые точки</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Клиенты</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Аренда</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hlButtons" stretch="0,0,0,1,0">
          <item>
           <widget class="QPushButton" name="m_pbAdd">
            <property name="text">
             <string>&amp;Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pbEdit">
            <property name="text">
             <string>&amp;Изменить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pbDelete">
            <property name="text">
             <string>&amp;Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_0">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="m_pbClear">
            <property name="text">
             <string>&amp;Очистить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_reportPage">
       <attribute name="title">
        <string>Отчёт</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblTemp">
          <property name="text">
           <string>Не реализовано</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="m_statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColouredLineEdit</class>
   <extends>QLineEdit</extends>
   <header>colouredLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_cbChoicesTable</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>m_swInputsData</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
